tests/Story.elm

```diff
@@ -104,7 +85,7 @@ suite =
                         queryFromStyledHtml (view storyFull)
                             |> Query.contains
                                 (List.map
-                                    (\image -> Html.img [ Html.Attributes.alt image.alt, Html.Attributes.src image.src ] [])
+                                    (\image -> Html.img [ Html.Attributes.alt image.alt, Html.Attributes.src image.src, Html.Attributes.class "_570ce029" ] [])
```

Comment:
Not ideal with the class hash but was after a quick fix. It may not change unless styles change - so probably ok.